src/tinode.js

```diff
@@ -1899,6 +1900,19 @@ export class Tinode {
     this.#send(pkt);
   }
 
+  call(topicName, seq, evt, payload) {
```

Comment:
Please add a js-doc comment.
It might be a good idea to give it a less generic name, like 'dial', 'videoCall'.